src/reference/asciidoc/amqp.adoc

```diff
@@ -3168,6 +3168,9 @@ For example, after your POJO has been converted, the `MessagePostProcessor` enab
 Starting with _version 1.4.2_, additional extension points have been added to the `RabbitTemplate` - `setBeforePublishPostProcessors()` and `setAfterReceivePostProcessors()`.
 The first enables a post processor to run immediately before sending to RabbitMQ. When using batching (see <<template-batching>>), this is invoked after the batch is assembled and before the batch is sent. The second is invoked immediately after a message is received.
 
+_version 2.1.4_ introduced `addBeforePublishPostProcessor()` and `addAfterReceivePostProcessor()` to the `RabbitTemplate` to allow appending new post processor to the list of before publish and after receive post processors respectively. Also there are methods provided to remove the post processors.
```

Comment:
I don't think that sentence should start like this.
Also don't use that `_` symbol to wrap version anymore.

Plus, please, follow "One sentence per line" recommendation: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line